types/html-renderer.d.ts

```diff
@@ -33,7 +34,17 @@ declare global {
   var ReportRenderer: typeof _ReportRenderer;
   var ReportUIFeatures: typeof _ReportUIFeatures;
   var Util: typeof _Util;
+  var Base64: typeof _Base64;
   var prepareLabData: typeof _prepareLabData;
+  var pako: typeof import('pako');
```

Comment:
this is kind of weird to add to our report types. Wdyt about `@ts-expect-error` the two lines in `base64.js` and let the base64 unit tests catch any errors in use?